extra/3-magic-8-ball.js

```diff
@@ -59,8 +66,26 @@ function shakeBall() {
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
 function checkAnswer(answer) {
-  //Write your code in here
+  let answerIteration;
+  for(let i = 0; i < answersArray.length; i++) {
+    if(answer === answersArray[i]) {
+      answerIteration = i;
+    }
+  }
+  //refactored my code below to make it into a ternary operator
+  return answerIteration < 5 ? "very positive" : answerIteration >=5 && answerIteration < 10 ? "positive"
+  : answerIteration >=10 && answerIteration < 15 ? "negative" : "very negative";
```

Comment:
**Style**

Please **Never** use nested ternary statements, **ever**

For the following multiplicity logic - not a real term, just a way I describe it.
Only use ternary for single branched out logic

For the code here, stick to using the following: 

Increase in (reading) complexity the way down

1. (A bunch of) `if` statements
2. Then, `switch` statements (or objects - key value pairs, if you know how to use them)
3. Lastly, `if...else` statements

Reply:
This was setting myself the challenge of seeing if I could convert it and understand nested ternary statements. Below is the code I used to work it out using If /else logic